brainscore_vision/benchmarks/maniquet2024/test.py

```diff
@@ -9,9 +9,12 @@
 import pytest
 from brainscore_vision import load_benchmark
 
```

Comment:
```suggestion
@pytest.mark.private_access
```
Loading benchmark requires S3 credentials